tests/network/l2_bridge/libl2bridge.py

```
@@ -236,18 +256,113 @@ def hot_plug_interface_and_set_address(
+        VMInterfaceStatusNotFoundError: If no interface with the expected MAC is found.
+    """
+    cmd = "ip -j addr show"
+    output = vm_console_run_commands(vm=vm, commands=[cmd], timeout=30)
+    raw_lines = output[cmd][1:]
+    json_line = next((line for line in raw_lines if line.strip()), "")
+    try:
+        guest_interfaces = json.loads(json_line)
+    except json.JSONDecodeError as err:
+        raise VMInterfaceStatusNotFoundError(
+            f"Failed to parse console JSON from VM {vm.name} for '{cmd}': {raw_lines}"
+        ) from err
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)